AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/ApkStore.kt

```diff
@@ -84,6 +84,34 @@ internal class ApkStore(
         }
     }
 
+    suspend fun uploadAdditionalApk(filePath: String): UploadedApk {
```

Comment:
yea i think this should be in Androidx.dev.

this code is making assumptions on the path we receive but ci-action is just a generic library. (e.g. it doesn't care or know about chrome).

this logic should be in androidx.dev and the only changes here should be about having ability to set additional files